data/brands/shop/mobile_phone.json

```diff
@@ -843,14 +843,14 @@
       }
     },
     {
-      "displayName": "TIM",
+      "displayName": "TIM Group",
       "id": "tim-a04a47",
-      "locationSet": {"include": ["br", "it"]},
+      "locationSet": {"include": ["br", "it", "sm"]},
       "tags": {
         "brand": "TIM",
         "brand:wikidata": "Q144617",
         "brand:wikipedia": "en:Gruppo TIM",
-        "name": "TIM",
+        "name": "TIM Group",
```

Comment:
Are these stores actually called "TIM Group", like on signs and stuff?  
The name is more what people commonly refer to the store as, not the official company name.